### PR TITLE
Vídeos e Documentos

### LINKED ISSUE #71: Detalhes da tecnologia - documentos
- [x] Integrar seção de documentos.

### PR DESCRIPTION
**Pull request by mtullyoc**
## Summary

Resolves #71 

simulator_screenshot_369614CB-553E-4C15-86FC-DB3A28BE923B

### CODE REVIEW

File: src/screens/Technology/components/Details/index.tsx [hunk 1]
```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
         presentationStyle="overFullScreen"
       />
 
-      {false && <S.CostSection>Vídeos</S.CostSection>}
+      {technology.attachments.documents.length ? (
```
Comment (mateus4k) [inline src/screens/Technology/components/Details/index.tsx]: Here we will have unexpected behavior. The length will be displayed on the screen. How to fix 👇 
```suggestion
      {!!technology.attachments.documents.length ? (
```

File: src/screens/Technology/components/Details/index.tsx [hunk 2]
```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
...
+        <>
+          <S.CostSection style={{ marginTop: 30 }}>Vídeos</S.CostSection>
+
+          {videos.map((video, idx) => (
+            <View key={idx} style={{ marginBottom: 0, backgroundColor: '#ddd' }}>
```
Comment (mateus4k) [inline src/screens/Technology/components/Details/index.tsx]: It's not a good practice use the array index as an item key. It's better to use a static identifier like the video id.

```suggestion
          {videos.map((video) => (
            <View key={video.videoId} style={{ marginBottom: 0, backgroundColor: '#ddd' }}>
```

https://reactjs.org/docs/lists-and-keys.html#keys

File: src/screens/Technology/components/Details/index.tsx [hunk 3]
```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
...
+              <YoutubePlayer
+                height={240}
+                videoId={video.videoId}
+              />
+            </View>
+          ))}
+        </>
+      ) : null}
+
+      {technology.attachments.documents.length ? (
```
Comment (mateus4k) [inline src/screens/Technology/components/Details/index.tsx]:
```suggestion
      {!!technology.attachments.documents.length ? (
```

File: src/screens/Technology/components/Details/index.tsx [hunk 4]
```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
...
+        <>
+          <S.CostSection style={{ marginTop: 30 }}>Documentos</S.CostSection>
+          {technology.attachments.documents.map((document, idx) => (
+            <TouchableOpacity
+              key={idx}
```
Comment (mateus4k) [inline src/screens/Technology/components/Details/index.tsx]: Check if `id` exists in `document` before changes, please
```suggestion
          {technology.attachments.documents.map((document, idx) => (
            <TouchableOpacity
              key={document.id}
```

File: src/screens/Technology/components/Details/index.tsx (hunk 2 above)
Comment (mtullyoc) [reply src/screens/Technology/components/Details/index.tsx]: NIce! Done.

**Pull request merged**